src/test/java/org/commcare/formplayer/tests/EndpointActionsTest.kt

```diff
@@ -1,23 +1,51 @@
 package org.commcare.formplayer.tests
 
+import org.commcare.formplayer.application.MenuController
+import org.commcare.formplayer.beans.menus.BaseResponseBean
 import org.commcare.formplayer.beans.menus.EntityListResponse
-import org.junit.jupiter.api.Assertions.*
+import org.commcare.formplayer.configuration.CacheConfiguration
+import org.commcare.formplayer.junit.*
+import org.commcare.formplayer.junit.request.SessionNavigationRequest
+import org.commcare.formplayer.utils.TestContext
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNull
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.RegisterExtension
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.web.servlet.MockMvc
 
 @WebMvcTest
-class EndpointActionsTest : BaseTestClass() {
+@Import(MenuController::class)
+@ContextConfiguration(classes = [TestContext::class, CacheConfiguration::class])
+@ExtendWith(InitializeStaticsExtension::class)
+class EndpointActionsTest {
```

Comment:
changes here are not related to PR and I end up doing it under a wrong assumption, but keeping the changes since it's still a good change to make.